RELNOTES.md

```diff
@@ -16,11 +23,24 @@
   version, if applicable. `tpaexec deploy` will then apply these changes
   to the servers in the cluster.
 
+- TPA-637 Allow a different Barman user when connecting to servers
+
 ### Minor changes
 
-- TPA-634 Support Oracle Linux 7 and 8 on the Docker platform
+- TPA-467 Explicit permisions when creating filesystem objects
```

Comment:
Should it add that this was for changing mode octals to strings, per Ansible `risky-octal` linting?